### ISSUE TITLE
reload causes address in use if not shutdown gracefully

### ISSUE THREAD

**Issue by kjaymiller** [render-engine/render-engine#300]
TLDR:
- OSError: [Errno 48] Address already in use
(.venv) render-engine-kjaymiller-theme[removes-SITE_URL-in-header-navigation*] % python -m render_engine serve --build docs.app:app --reload

```shell
(.venv) docs[removes-SITE_URL-in-header-navigation*] % python -m render_engine serve --build app:app --reload      
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ /Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-theme/.venv/lib/pyt │
│ hon3.11/site-packages/render_engine/cli.py:295 in serve                                          │
│                                                                                                  │
│   292 │                                                                                          │
│   293 │                                                                                          │
│   294 │   if module_site:                                                                        │
│ ❱ 295 │   │   app = get_app(module_site)                                                         │
│   296 │   │   app.render()                                                                       │
│   297 │                                                                                          │
│   298 │   if not directory:                                                                      │
│                                                                                                  │
│ ╭──────── locals ─────────╮                                                                      │
│ │   directory = None      │                                                                      │
│ │ module_site = 'app:app' │                                                                      │
│ │        port = 8000      │                                                                      │
│ │      reload = True      │                                                                      │
│ ╰─────────────────────────╯                                                                      │
│                                                                                                  │
│ /Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-theme/.venv/lib/pyt │
│ hon3.11/site-packages/render_engine/cli.py:25 in get_app                                         │
│                                                                                                  │
│    22 │   """Split the site module into a module and a class name"""                             │
│    23 │   sys.path.insert(0, ".")                                                                │
│    24 │   import_path, app_name = module_site.split(":", 1)                                      │
│ ❱  25 │   importlib.import_module(import_path)                                                   │
│    26 │   return getattr(sys.modules[import_path], app_name)                                     │
│    27                                                                                            │
│    28                                                                                            │
│                                                                                                  │
│ ╭──────── locals ─────────╮                                                                      │
│ │    app_name = 'app'     │                                                                      │
│ │ import_path = 'app'     │                                                                      │
│ │ module_site = 'app:app' │                                                                      │
│ ╰─────────────────────────╯                                                                      │
│                                                                                                  │
│ /Users/jaymiller/.pyenv/versions/3.11.4/lib/python3.11/importlib/__init__.py:126 in              │
│ import_module                                                                                    │
│                                                                                                  │
│   123 │   │   │   if character != '.':                                                           │
│   124 │   │   │   │   break                                                                      │
│   125 │   │   │   level += 1                                                                     │
│ ❱ 126 │   return _bootstrap._gcd_import(name[level:], package, level)                            │
│   127                                                                                            │
│   128                                                                                            │
│   129 _RELOADING = {}                                                                            │
│                                                                                                  │
│ ╭──── locals ─────╮                                                                              │
│ │   level = 0     │                                                                              │
│ │    name = 'app' │                                                                              │
│ │ package = None  │                                                                              │
│ ╰─────────────────╯                                                                              │
│ in _gcd_import:1204                                                                              │
│ ╭──── locals ─────╮                                                                              │
│ │   level = 0     │                                                                              │
│ │    name = 'app' │                                                                              │
│ │ package = None  │                                                                              │
│ ╰─────────────────╯                                                                              │
│ in _find_and_load:1176                                                                           │
│ ╭──────────────────── locals ─────────────────────╮                                              │
│ │ import_ = <function _gcd_import at 0x10468fd80> │                                              │
│ │  module = <object object at 0x1046c4050>        │                                              │
│ │    name = 'app'                                 │                                              │
│ ╰─────────────────────────────────────────────────╯                                              │
│ in _find_and_load_unlocked:1147                                                                  │
│ ╭─────────────────────────────────────────── locals ───────────────────────────────────────────╮ │
│ │     import_ = <function _gcd_import at 0x10468fd80>                                          │ │
│ │        name = 'app'                                                                          │ │
│ │      parent = ''                                                                             │ │
│ │ parent_spec = None                                                                           │ │
│ │        path = None                                                                           │ │
│ │        spec = ModuleSpec(name='app', loader=<_frozen_importlib_external.SourceFileLoader     │ │
│ │               object at 0x106a0ad10>,                                                        │ │
│ │               origin='/Users/jaymiller/render-engine/render_engine_themes/render-engine-kja… │ │
│ ╰──────────────────────────────────────────────────────────────────────────────────────────────╯ │
│ in _load_unlocked:690                                                                            │
│ ╭─────────────────────────────────────────── locals ───────────────────────────────────────────╮ │
│ │ module = <module 'app' from                                                                  │ │
│ │          '/Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-them… │ │
│ │   spec = ModuleSpec(name='app', loader=<_frozen_importlib_external.SourceFileLoader object   │ │
│ │          at 0x106a0ad10>,                                                                    │ │
│ │          origin='/Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymill… │ │
│ ╰──────────────────────────────────────────────────────────────────────────────────────────────╯ │
│ in exec_module:940                                                                               │
│ ╭─────────────────────────────────────────── locals ───────────────────────────────────────────╮ │
│ │   code = <code object <module> at 0x136a05ba0, file                                          │ │
│ │          "/Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-them… │ │
│ │          line 1>                                                                             │ │
│ │ module = <module 'app' from                                                                  │ │
│ │          '/Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-them… │ │
│ │   self = <_frozen_importlib_external.SourceFileLoader object at 0x106a0ad10>                 │ │
│ ╰──────────────────────────────────────────────────────────────────────────────────────────────╯ │
│ in _call_with_frames_removed:241                                                                 │
│ ╭─────────────────────────────────────────── locals ───────────────────────────────────────────╮ │
│ │ args = (                                                                                     │ │
│ │        │   <code object <module> at 0x136a05ba0, file                                        │ │
│ │        "/Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-theme/… │ │
│ │        line 1>,                                                                              │ │
│ │        │   {                                                                                 │ │
│ │        │   │   '__name__': 'app',                                                            │ │
│ │        │   │   '__doc__': None,                                                              │ │
│ │        │   │   '__package__': '',                                                            │ │
│ │        │   │   '__loader__': <_frozen_importlib_external.SourceFileLoader object at          │ │
│ │        0x106a0ad10>,                                                                         │ │
│ │        │   │   '__spec__': ModuleSpec(name='app',                                            │ │
│ │        loader=<_frozen_importlib_external.SourceFileLoader object at 0x106a0ad10>,           │ │
│ │        origin='/Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller… │ │
│ │        │   │   '__file__':                                                                   │ │
│ │        '/Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-the'+1… │ │
│ │        │   │   '__cached__':                                                                 │ │
│ │        '/Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-the'+3… │ │
│ │        │   │   '__builtins__': {                                                             │ │
│ │        │   │   │   '__name__': 'builtins',                                                   │ │
│ │        │   │   │   '__doc__': 'Built-in functions, types, exceptions, and other              │ │
│ │        objects.\n\nThis module provides '+346,                                               │ │
│ │        │   │   │   '__package__': '',                                                        │ │
│ │        │   │   │   '__loader__': <class '_frozen_importlib.BuiltinImporter'>,                │ │
│ │        │   │   │   '__spec__': ModuleSpec(name='builtins', loader=<class                     │ │
│ │        '_frozen_importlib.BuiltinImporter'>, origin='built-in'),                             │ │
│ │        │   │   │   '__build_class__': <built-in function __build_class__>,                   │ │
│ │        │   │   │   '__import__': <built-in function __import__>,                             │ │
│ │        │   │   │   'abs': <built-in function abs>,                                           │ │
│ │        │   │   │   'all': <built-in function all>,                                           │ │
│ │        │   │   │   'any': <built-in function any>,                                           │ │
│ │        │   │   │   ... +147                                                                  │ │
│ │        │   │   },                                                                            │ │
│ │        │   │   'Collection': <class 'render_engine.collection.Collection'>,                  │ │
│ │        │   │   'Page': <class 'render_engine.page.Page'>,                                    │ │
│ │        │   │   ... +5                                                                        │ │
│ │        │   }                                                                                 │ │
│ │        )                                                                                     │ │
│ │    f = <built-in function exec>                                                              │ │
│ │ kwds = {}                                                                                    │ │
│ ╰──────────────────────────────────────────────────────────────────────────────────────────────╯ │
│                                                                                                  │
│ /Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-theme/docs/app.py:5 │
│ 3 in <module>                                                                                    │
│                                                                                                  │
│   50 │   has_archive = True                                                                      │
│   51 │   archive_template = "blog_list.html"                                                     │
│   52                                                                                             │
│ ❱ 53 @app.page                                                                                   │
│   54 class Index(Page):                                                                          │
│   55 │   template = "page.html"                                                                  │
│   56 │   title = ""                                                                              │
│                                                                                                  │
│ ╭─────────────────────────────────────────── locals ───────────────────────────────────────────╮ │
│ │                app = <render_engine.site.Site object at 0x1069d8f10>                         │ │
│ │         Collection = <class 'render_engine.collection.Collection'>                           │ │
│ │               Docs = Collection: Collection                                                  │ │
│ │         kjaymiller = Theme(                                                                  │ │
│ │                      │   loader=<jinja2.loaders.PackageLoader object at 0x106a0be90>,        │ │
│ │                      │   filters={                                                           │ │
│ │                      │   │   'theme_gradient_to': <function get_theme_gradient_to at         │ │
│ │                      0x106a051c0>,                                                           │ │
│ │                      │   │   'theme_color_value': <function get_color_string_value at        │ │
│ │                      0x106a056c0>                                                            │ │
│ │                      │   },                                                                  │ │
│ │                      │   plugins=[<class 'render_engine_tailwindcss.plugins.TailwindCSS'>],  │ │
│ │                      │                                                                       │ │
│ │                      static_dir=PosixPath('/Users/jaymiller/render-engine/render_engine_the… │ │
│ │                      )                                                                       │ │
│ │ MarkdownPageParser = <class 'render_engine.parsers.markdown.MarkdownPageParser'>             │ │
│ │               Page = <class 'render_engine.page.Page'>                                       │ │
│ │               Site = <class 'render_engine.site.Site'>                                       │ │
│ ╰──────────────────────────────────────────────────────────────────────────────────────────────╯ │
│                                                                                                  │
│ /Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-theme/.venv/lib/pyt │
│ hon3.11/site-packages/render_engine/site.py:166 in page                                          │
│                                                                                                  │
│   163 │   │   site.page(About) # also works                                                      │
│   164 │   │   ```                                                                                │
│   165 │   │   """                                                                                │
│ ❱ 166 │   │   page = Page()                                                                      │
│   167 │   │   page.title = page._title # Expose _title to the user through `title`               │
│   168 │   │                                                                                      │
│   169 │   │   # copy the plugin manager, removing any plugins that the page has ignored          │
│                                                                                                  │
│ ╭──────────────────────── locals ────────────────────────╮                                       │
│ │ Page = <class 'app.Index'>                             │                                       │
│ │ self = <render_engine.site.Site object at 0x1069d8f10> │                                       │
│ ╰────────────────────────────────────────────────────────╯                                       │
│                                                                                                  │
│ /Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-theme/.venv/lib/pyt │
│ hon3.11/site-packages/render_engine/page.py:154 in __init__                                      │
│                                                                                                  │
│   151 │   │                                                                                      │
│   152 │   │   # Parse Content from the Content Path or the Content                               │
│   153 │   │   if content_path := (content_path or getattr(self, "content_path", None)):          │
│ ❱ 154 │   │   │   attrs, self.content = self.Parser.parse_content_path(content_path)             │
│   155 │   │                                                                                      │
│   156 │   │   elif content := (content or getattr(self, "content", None)):                       │
│   157 │   │   │   attrs, self.content = self.Parser.parse_content(content)                       │
│                                                                                                  │
│ ╭────────── locals ──────────╮                                                                   │
│ │      content = None        │                                                                   │
│ │ content_path = 'README.md' │                                                                   │
│ │       Parser = None        │                                                                   │
│ │         self = <Page: >    │                                                                   │
│ ╰────────────────────────────╯                                                                   │
│                                                                                                  │
│ /Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-theme/.venv/lib/pyt │
│ hon3.11/site-packages/render_engine/parsers/base_parsers.py:30 in parse_content_path             │
│                                                                                                  │
│   27 │   │   │   │   The path to the file that will be used to generate the Page's `content`.    │
│   28 │   │   │   │   Should be a valid path to a file or a url.                                  │
│   29 │   │   """                                                                                 │
│ ❱ 30 │   │   return parse_content(pathlib.Path(content_path).read_text())                        │
│   31 │                                                                                           │
│   32 │   @staticmethod                                                                           │
│   33 │   def parse_content(content: str) -> tuple[dict, str]:                                    │
│                                                                                                  │
│ ╭────────── locals ──────────╮                                                                   │
│ │ content_path = 'README.md' │                                                                   │
│ ╰────────────────────────────╯                                                                   │
│                                                                                                  │
│ /Users/jaymiller/.pyenv/versions/3.11.4/lib/python3.11/pathlib.py:1058 in read_text              │
│                                                                                                  │
│   1055 │   │   Open the file in text mode, read it, and close the file.                          │
│   1056 │   │   """                                                                               │
│   1057 │   │   encoding = io.text_encoding(encoding)                                             │
│ ❱ 1058 │   │   with self.open(mode='r', encoding=encoding, errors=errors) as f:                  │
│   1059 │   │   │   return f.read()                                                               │
│   1060 │                                                                                         │
│   1061 │   def write_bytes(self, data):                                                          │
│                                                                                                  │
│ ╭───────────── locals ──────────────╮                                                            │
│ │ encoding = 'locale'               │                                                            │
│ │   errors = None                   │                                                            │
│ │     self = PosixPath('README.md') │                                                            │
│ ╰───────────────────────────────────╯                                                            │
│                                                                                                  │
│ /Users/jaymiller/.pyenv/versions/3.11.4/lib/python3.11/pathlib.py:1044 in open                   │
│                                                                                                  │
│   1041 │   │   """                                                                               │
│   1042 │   │   if "b" not in mode:                                                               │
│   1043 │   │   │   encoding = io.text_encoding(encoding)                                         │
│ ❱ 1044 │   │   return io.open(self, mode, buffering, encoding, errors, newline)                  │
│   1045 │                                                                                         │
│   1046 │   def read_bytes(self):                                                                 │
│   1047 │   │   """                                                                               │
│                                                                                                  │
│ ╭────────────── locals ──────────────╮                                                           │
│ │ buffering = -1                     │                                                           │
│ │  encoding = 'locale'               │                                                           │
│ │    errors = None                   │                                                           │
│ │      mode = 'r'                    │                                                           │
│ │   newline = None                   │                                                           │
│ │      self = PosixPath('README.md') │                                                           │
│ ╰────────────────────────────────────╯                                                           │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
FileNotFoundError: [Errno 2] No such file or directory: 'README.md'
(.venv) docs[removes-SITE_URL-in-header-navigation*] % 
(.venv) docs[removes-SITE_URL-in-header-navigation*] % cd ..                                                       
(.venv) render-engine-kjaymiller-theme[removes-SITE_URL-in-header-navigation*] % python -m render_engine serve --build docs.app:app --reload
Loading Pre-Build Plugins  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00
AddingRoute: index         ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0% -:--:--
Loading Post-Build Plugins ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0% -:--:--
Watching for changes...
Serving docs/output
Spawning server on http://127.0.0.1:8000
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ /Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-theme/.venv/lib/pyt │
│ hon3.11/site-packages/render_engine/cli.py:321 in serve                                          │
│                                                                                                  │
│   318 │   │   handler._server.serve_forever()                                                    │
│   319 │   else:                                                                                  │
│   320 │   │   console.print("Watching for changes...")                                           │
│ ❱ 321 │   │   handler.watch()                                                                    │
│   322                                                                                            │
│   323 def cli():                                                                                 │
│   324 │   app()                                                                                  │
│                                                                                                  │
│ ╭───────────────────────────────────── locals ──────────────────────────────────────╮            │
│ │            app = <render_engine.site.Site object at 0x106ad3710>                  │            │
│ │        console = <console width=131 ColorSystem.TRUECOLOR>                        │            │
│ │      directory = 'docs/output'                                                    │            │
│ │        handler = <render_engine.watcher.event.RegExHandler object at 0x106529350> │            │
│ │    module_site = 'docs.app:app'                                                   │            │
│ │           port = 8000                                                             │            │
│ │         reload = True                                                             │            │
│ │ server_address = ('127.0.0.1', 8000)                                              │            │
│ ╰───────────────────────────────────────────────────────────────────────────────────╯            │
│                                                                                                  │
│ /Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-theme/.venv/lib/pyt │
│ hon3.11/site-packages/render_engine/watcher/event.py:127 in watch                                │
│                                                                                                  │
│   124 │   │                                                                                      │
│   125 │   │   observer = Observer()                                                              │
│   126 │   │   observer.schedule(self, self.dir_to_watch, recursive=True)                         │
│ ❱ 127 │   │   self.start_server()                                                                │
│   128 │   │   observer.start()                                                                   │
│   129 │   │                                                                                      │
│   130 │   │   try:                                                                               │
│                                                                                                  │
│ ╭────────────────────────────────── locals ───────────────────────────────────╮                  │
│ │ observer = <FSEventsObserver(Thread-2, initial daemon)>                     │                  │
│ │     self = <render_engine.watcher.event.RegExHandler object at 0x106529350> │                  │
│ ╰─────────────────────────────────────────────────────────────────────────────╯                  │
│                                                                                                  │
│ /Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-theme/.venv/lib/pyt │
│ hon3.11/site-packages/render_engine/watcher/event.py:84 in start_server                          │
│                                                                                                  │
│    81 │                                                                                          │
│    82 │   def start_server(self):                                                                │
│    83 │   │   console.print(f"[bold green]Spawning server on http://{self.server_address[0]}:{   │
│ ❱  84 │   │   self._server = spawn_server(self.server_address, self.dir_to_serve[0])             │
│    85 │   │   self._thread = threading.Thread(target=self._server.serve_forever)                 │
│    86 │   │   self._thread.start()                                                               │
│    87                                                                                            │
│                                                                                                  │
│ ╭──────────────────────────────── locals ─────────────────────────────────╮                      │
│ │ self = <render_engine.watcher.event.RegExHandler object at 0x106529350> │                      │
│ ╰─────────────────────────────────────────────────────────────────────────╯                      │
│                                                                                                  │
│ /Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-theme/.venv/lib/pyt │
│ hon3.11/site-packages/render_engine/watcher/event.py:31 in spawn_server                          │
│                                                                                                  │
│    28 │                                                                                          │
│    29 │   def _httpd() -> ThreadingHTTPServer:                                                   │
│    30 │   │   return ThreadingHTTPServer(server_address, _RequestHandler)                        │
│ ❱  31 │   return _httpd()                                                                        │
│    32                                                                                            │
│    33                                                                                            │
│    34 class RegExHandler(RegexMatchingEventHandler):                                             │
│                                                                                                  │
│ ╭─────────────────────────────────────────── locals ───────────────────────────────────────────╮ │
│ │          _httpd = <function spawn_server.<locals>._httpd at 0x106b26980>                     │ │
│ │ _RequestHandler = <class                                                                     │ │
│ │                   'render_engine.watcher.event.spawn_server.<locals>._RequestHandler'>       │ │
│ │       directory = 'docs/output'                                                              │ │
│ │  server_address = ('127.0.0.1', 8000)                                                        │ │
│ ╰──────────────────────────────────────────────────────────────────────────────────────────────╯ │
│                                                                                                  │
│ /Users/jaymiller/render-engine/render_engine_themes/render-engine-kjaymiller-theme/.venv/lib/pyt │
│ hon3.11/site-packages/render_engine/watcher/event.py:30 in _httpd                                │
│                                                                                                  │
│    27 │   │   │   super().__init__(*args, directory=directory, **kwargs)                         │
│    28 │                                                                                          │
│    29 │   def _httpd() -> ThreadingHTTPServer:                                                   │
│ ❱  30 │   │   return ThreadingHTTPServer(server_address, _RequestHandler)                        │
│    31 │   return _httpd()                                                                        │
│    32                                                                                            │
│    33                                                                                            │
│                                                                                                  │
│ ╭─────────────────────────────────────────── locals ───────────────────────────────────────────╮ │
│ │ _RequestHandler = <class                                                                     │ │
│ │                   'render_engine.watcher.event.spawn_server.<locals>._RequestHandler'>       │ │
│ │  server_address = ('127.0.0.1', 8000)                                                        │ │
│ ╰──────────────────────────────────────────────────────────────────────────────────────────────╯ │
│                                                                                                  │
│ /Users/jaymiller/.pyenv/versions/3.11.4/lib/python3.11/socketserver.py:456 in __init__           │
│                                                                                                  │
│   453 │   │   │   │   │   │   │   │   │   self.socket_type)                                      │
│   454 │   │   if bind_and_activate:                                                              │
│   455 │   │   │   try:                                                                           │
│ ❱ 456 │   │   │   │   self.server_bind()                                                         │
│   457 │   │   │   │   self.server_activate()                                                     │
│   458 │   │   │   except:                                                                        │
│   459 │   │   │   │   self.server_close()                                                        │
│                                                                                                  │
│ ╭─────────────────────────────────────────── locals ───────────────────────────────────────────╮ │
│ │   bind_and_activate = True                                                                   │ │
│ │ RequestHandlerClass = <class                                                                 │ │
│ │                       'render_engine.watcher.event.spawn_server.<locals>._RequestHandler'>   │ │
│ │                self = <http.server.ThreadingHTTPServer object at 0x106c24410>                │ │
│ │      server_address = ('127.0.0.1', 8000)                                                    │ │
│ ╰──────────────────────────────────────────────────────────────────────────────────────────────╯ │
│                                                                                                  │
│ /Users/jaymiller/.pyenv/versions/3.11.4/lib/python3.11/http/server.py:136 in server_bind         │
│                                                                                                  │
│    133 │                                                                                         │
│    134 │   def server_bind(self):                                                                │
│    135 │   │   """Override server_bind to store the server name."""                              │
│ ❱  136 │   │   socketserver.TCPServer.server_bind(self)                                          │
│    137 │   │   host, port = self.server_address[:2]                                              │
│    138 │   │   self.server_name = socket.getfqdn(host)                                           │
│    139 │   │   self.server_port = port                                                           │
│                                                                                                  │
│ ╭──────────────────────────── locals ────────────────────────────╮                               │
│ │ self = <http.server.ThreadingHTTPServer object at 0x106c24410> │                               │
│ ╰────────────────────────────────────────────────────────────────╯                               │
│                                                                                                  │
│ /Users/jaymiller/.pyenv/versions/3.11.4/lib/python3.11/socketserver.py:472 in server_bind        │
│                                                                                                  │
│   469 │   │   │   self.socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)              │
│   470 │   │   if self.allow_reuse_port and hasattr(socket, "SO_REUSEPORT"):                      │
│   471 │   │   │   self.socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEPORT, 1)              │
│ ❱ 472 │   │   self.socket.bind(self.server_address)                                              │
│   473 │   │   self.server_address = self.socket.getsockname()                                    │
│   474 │                                                                                          │
│   475 │   def server_activate(self):                                                             │
│                                                                                                  │
│ ╭──────────────────────────── locals ────────────────────────────╮                               │
│ │ self = <http.server.ThreadingHTTPServer object at 0x106c24410> │                               │
│ ╰────────────────────────────────────────────────────────────────╯                               │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
OSError: [Errno 48] Address already in use
(.venv) render-engine-kjaymiller-theme[removes-SITE_URL-in-header-navigation*] % python -m render_engine serve --build docs.app:app --reload
```

**Comment by kjaymiller** [comment]
Fixed in #307
